app/src/main/java/org/openobservatory/ooniprobe/fragment/PreferenceGlobalFragment.java

```diff
@@ -1,5 +1,6 @@
 package org.openobservatory.ooniprobe.fragment;
 
+import android.content.SharedPreferences;
```

Comment:
@lorenzoPrimi this import seems stray. There are otherwise no other changes in this file.